RichText-web-module/src/main/webapp/resources/celRteJS/tinyMCE4/loadTinyMCE-async.js

```diff
@@ -65,8 +65,8 @@
   /**
    * loading in struct layout editor
    **/
-  (function(structManager){
-    console.log('loadTinyMCE async: start');
+  const initCelRTE4InStructEditor = function(event) {
+    const structManager = event?.detail?.structEditorManager ?? window.celStructEditorManager;
```

Comment:
we have had issues with Safari, because it is supporting the optional chaining operator only since March 2020. Please use it only in babbled mjs files.